tensorflow/python/ops/image_ops_impl.py

```diff
@@ -3678,6 +3678,22 @@ def extract_glimpse(
 
   Returns:
     A `Tensor` of type `float32`.
+    
+  Usage Example:
+    ```python
+    import tensorflow as tf
+    BATCH_SIZE = 2
+    IMAGE_HEIGHT = 256
+    IMAGE_WIDTH = 256
+    CHANNELS = 3
+    GLIMPSE_SIZE = (24, 24)
+    image = tf.random.normal(shape=(BATCH_SIZE, IMAGE_HEIGHT, 
+    IMAGE_WIDTH, CHANNELS) )
+    offsets_variable = tf.random.uniform(shape=(BATCH_SIZE, 2))
+    output = tf.image.extract_glimpse(image, size=GLIMPSE_SIZE, 
+    offsets=offsets_variable)
+    print(output.shape)
```

Comment:
It'd be nice if we could show something here at all. If I read this code and not execute it I get almost no information out of it (other than the requirements on the input shapes of the op).

Can we pick a known small one-channel image and show what a glimpse is?

Reply:
Sure let me add that